src/main/java/QueryServlet.java

```diff
@@ -111,15 +117,20 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
     if (!queryToDataRowGeneric.containsKey(action)) {
       // We don't have information on this action
-      response.sendError(
-          HttpServletResponse.SC_NOT_IMPLEMENTED, "We do not support this visualization yet.");
+      response.setStatus(HttpServletResponse.SC_NOT_IMPLEMENTED);
+      response.setContentType("application/json;");
+      response.getWriter().println(sendError("We do not support this visualization yet."));
       return;
     } else if (!queryToDataRowGeneric.get(action).containsKey(personType)) {
       // This action doesn't make sense with this type of person,
       // or the census doesn't keep data on it that we could find
-      response.sendError(
-          HttpServletResponse.SC_BAD_REQUEST,
-          "This query is not supported by census data. Try asking a more general one.");
+      response.setStatus(HttpServletResponse.SC_NOT_IMPLEMENTED);
+      response.setContentType("application/json;");
+      response
+          .getWriter()
+          .println(
+              sendError(
+                  "This query is not supported by census data. Try asking a more general one."));
```

Comment:
Is there a reason not to do this in two lines? something like, 
response.getWriter().println(sendError(
"This query is not ....

Reply:
Yeah, I had it in two lines but the linter asked for it to be like this, not really sure what the benefit is